features/guidance/bicycle-sliproads.feature

```diff
@@ -0,0 +1,33 @@
+@routing @bicycle @mode
+Feature: Bike - Mode flag
+
+	Background:
+		Given the profile "bicycle"
+        Given a grid size of 5 meters
+
+    Scenario: Bike Sliproad
+        Given the node map
+            """
+                  i
+            a b - c-d
+                ` |
+                g-e-h
+                  |
+                  |
+                  f
+            """
+
+        And the nodes
+            | node | highway         |
+            | c    | traffic_signals |
+
+        And the ways
+            | nodes | highway   | name   | oneway:bicycle |
+            | abcd  | cycleway  | street |                |
+            | eb    | path      |        | yes            |
+            | icef  | tertiary  | road   |                |
+            | geh   | secondary | street |                |
+
+        When I route I should get
+            | waypoints | route                           | turns                               |
+            | a,f       | street,{highway:path},road,road | depart,turn right,turn right,arrive |
```

Comment:
Unrelated but I'm just seeing this here: is there a reason for us to still encode the name with the highway tag in _the default profile_? If people want to do this, fine, feel free to change the profile.